util/memory/tracker.go

```diff
@@ -354,8 +366,11 @@ func (t *Tracker) Consume(bs int64) {
 	if bs == 0 {
 		return
 	}
-	var rootExceed, rootExceedForSoftLimit *Tracker
+	var rootExceed, rootExceedForSoftLimit, sessionTracker *Tracker
```

Comment:
This variable is somewhat strange, I think we can remove this var, and check `tracker.IsSession` below.